sdk/storage/azdatalake/file/models.go

```diff
@@ -230,6 +237,15 @@ func (o *FlushDataOptions) format(offset int64) (*generated.PathClientFlushDataO
 			cpkInfoOpts.EncryptionKeySHA256 = o.CPKInfo.EncryptionKeySHA256
 			cpkInfoOpts.EncryptionAlgorithm = o.CPKInfo.EncryptionAlgorithm
 		}
+		if o.LeaseAction != nil {
+			flushDataOpts.LeaseAction = o.LeaseAction
+		}
+		if o.LeaseDuration != nil {
+			flushDataOpts.LeaseDuration = o.LeaseDuration
+		}
+		if o.ProposedLeaseID != nil {
+			flushDataOpts.ProposedLeaseID = o.ProposedLeaseID
+		}
```

Comment:
You can directly assign the values to flushDataOpts instead of having conditional check like,
```
flushDataOpts.LeaseAction = o.LeaseAction
flushDataOpts.LeaseDuration = o.LeaseDuration
flushDataOpts.ProposedLeaseID = o.ProposedLeaseID
```

Reply:
resolved